.gitignore

```diff
@@ -0,0 +1,22 @@
+# OS and Editor files
```

Comment:
I laugh because I totally just copied from my global gitignore. 😁 

I want more green so I'm gonna just PR the git ignore as a separate thing. 👍